build.sbt

```diff
@@ -387,6 +387,14 @@ lazy val compilerBridge: Project = (project in internalPath / "compiler-bridge")
     publishLocal := publishLocal.dependsOn(cleanSbtBridge).value,
     altPublishSettings,
     mimaSettings,
+    mimaBinaryIssueFilters ++= Vector(
```

Comment:
I think we should follow the scalac standard of having this in an independent file. I don't like having this on the build.sbt.

Reply:
Could you elaborate why?

Reply:
I can think of two reasons:
  
* Every time I change something, It makes me reload the build.
* A file inside a project's source directories is clearer to find and read.

Reply:
Done in 505ac2e